### ISSUE TITLE
How to Use?

### ISSUE THREAD

**Issue by Appolonius001** [fkunn1326/openpose-editor#11]
I create and adjust my pose,
send to controlnet,
select and activate ControlNet
select openpose for model and preprocessor
enter a prompt and hit generate
the render looks nothing like the pose i created

**Comment by jparmstr** [comment]
Do not select anything for the Preprocessor. You've already got your "preprocessed" image that you created with this extension.

**Comment by runmax4** [comment]
Yes, do not select anything for the Preprocessor, but should select openpose for model.
